hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NetworkTopologyServlet.java

```diff
@@ -90,7 +90,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
    * @param leaves leaves nodes under base scope
    * @param format the response format
    */
-  public void printTopology(PrintStream stream, List<Node> leaves,
+  private void printTopology(PrintStream stream, List<Node> leaves,
```

Comment:
Why not just override?

Reply:
> Why not just override?

Because the arguments of the two methods are a little different, I'll fix it. Thank you for pointing that out.